docs/DESIGN.md

```diff
@@ -0,0 +1,33 @@
+# Patchwork's Design Principles
+
+This document describes the design goals and philosophy of Patchwork. Features of and changes to Patchwork should always aim to follow these principles. If we really want to add a feature/make a change to Patchwork that goes against these principles, we should think about changing the principles first, rather than ignoring them and making the change anyway.
+
+This document should be version tracked, and any change to it should go through the [advice process](http://www.reinventingorganizationswiki.com/Decision_Making) with the stakeholders listed below.
+
+### Stakeholders
+* Patchwork contributors
+* Patchwork users
+* The broader Scuttlebutt community
+* People outside the Scuttlebutt community who might be affected
+
+## Design goals
+
+### Goals and scope: Patchwork should...
```

Comment:
Note to self: add something about healthy design

Reply:
How about replacing the whole section with this:

### Goals and scope: Patchwork should...
* Follow the [Scuttlebutt principles](https://www.scuttlebutt.nz/principles/)
* Be a client that people can feel confident recommending to their friends who aren't using Scuttlebutt
* Be safe (privacy, blocking, etc)
* Be usable (not confusing or broken; clear explanations of new concepts)
* Be healthy (non-addictive, calm, consentful)
* Be accessible (to the extent practical -- some a11y features are hard to retrofit into the existing code and we will build the next client with this in mind)
* Be a fun client to use -- users should enjoy using it and, in a positive way, be tempted to try other Scuttlebutt clients
* Look good

Reply:
Awesome, thanks! Added this (but removed the comment about the next client since we don't control that)